Improve "How to" guides (#79)
OpenTermsArchive/docs: content/navigate-history.en.md

  ![One GitHub Privacy Policy change with source diff view](/images/navigate-history/source-diff.png)

- You can choose from two types of display with the icons in the grey bar above the document. The first one (which is also the default one), named _source diff_ displays the previous version and the next one [side by side](https://github.com/OpenTermsArchive/demo-versions/commit/e9a781797041a6b593967ba9e7bb2c7404390e76). The second one, named _rich diff_ displays all the changes [in a single document](https://github.com/OpenTermsArchive/demo-versions/commit/e9a781797041a6b593967ba9e7bb2c7404390e76?short_path=060f2c2#diff-060f2c2cc43c2415e0d388f0061c37472277e76eafc9c0df269713b150a52909). In this view, beyond green and red, the yellow color shows modified paragraphs. Be careful, this display does not show some changes such as hyperlinks and text style's changes:

  ![One GitHub Privacy Policy change with rich diff view](/images/navigate-history/rich-diff.png)

- - -

## Notes

> - For long documents, unchanged paragraphs will not be displayed by default. You can manually make them appear by clicking on the small arrows just above or just below the displayed paragraphs.

Reviewer: An image of the arrows could be helpful.

Reviewer 2: Since this is aimed at showing GitHub's icons, we can use [Primer Octicons](https://primer.style/design/foundations/icons/unfold-24) 🙂

Author: I'va already added screenshots

---
title: "Browsing through terms"
weight: 10
---

# Browsing through terms

Every collection offers a public database of versions they recorded. 

> For this guide, we will use the example of the demo collection. This collection publishes its history on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.

Reviewer 2: I would suggest capitalising the name of the collection, in order to make it easier to understand we are talking about a collection called “Demo” and not just “some random demo collection”.

```suggestion
For this guide, we will use the example of the Demo collection. The terms of this collection are published on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.
```

> - From the [repository page](https://github.com/OpenTermsArchive/demo-versions), open the folder of the service of your choice by clicking on it. For example, [GitHub](https://github.com/OpenTermsArchive/demo-versions/tree/main/GitHub):

Reviewer 2: > I would have preferred that we use Open Terms Archive as an example. It's okay to keep GitHub now that all the images have been done but, for future examples, the most stable example we can give is Open Terms Archive.

Author: I did it with Open Terms Terms at first but I encountered problems because our files did not changed and we did not have history. So I re do everything with GitHub…

  ![Demo-versions repository services list](/images/navigate-history/demo-repository.png)

> - You will see the set of documents tracked for that service, now click on the document of your choice. For example, [GitHub's Privacy Policy](https://github.com/OpenTermsArchive/demo-versions/blob/main/GitHub/Privacy%20Policy.md):

Reviewer 2: ```suggestion
- You will see the set of terms tracked for that service, now click on the terms of your choice. For example, [GitHub's Privacy Policy](https://github.com/OpenTermsArchive/demo-versions/blob/main/GitHub/Privacy%20Policy.md):
```

  ![GitHub terms list](/images/navigate-history/github-terms.png)

> - The latest version will be displayed. To view the history of changes made to this document, click on _History_ at the top right of the document:

Reviewer 2: > This is correct, but let's make it even less ambiguous 🙂 

```suggestion
- The most recent version will be displayed. To view the history of changes made to these terms, click on _History_ at the top right:
```

We could also use Primer Octicon's [history icon](https://primer.style/design/foundations/icons/history-24) since it's coming from GitHub.

  ![GitHub Privacy Policy](/images/navigate-history/github-privacy-policy.png)

> - The changes are ordered ante-chronologically. For example, [GitHub Privacy Policy history](https://github.com/OpenTermsArchive/demo-versions/commits/main/GitHub/Privacy%20Policy.md). Click on a change title to see its contents:

Reviewer 2: > I'm not sure which is clearer.
```suggestion
- The changes are presented in reverse chronological order. For example, [GitHub Privacy Policy history](https://github.com/OpenTermsArchive/demo-versions/commits/main/GitHub/Privacy%20Policy.md). Click on a change title to see its contents:
```

  ![GitHub Privacy Policy history](/images/navigate-history/privacy-policy-history.png)

> - The red colour shows deleted elements and the green colour shows added elements. For example, in this version of [GitHub Privacy Policy](https://github.com/OpenTermsArchive/demo-versions/commit/e9a781797041a6b593967ba9e7bb2c7404390e76), you can see in red the deletion of informations about GitHub data protection officer .

Reviewer 2: ```suggestion
- The red colour shows deleted words and the green colour shows added words. For example, in this change of [GitHub Privacy Policy](https://github.com/OpenTermsArchive/demo-versions/commit/e9a781797041a6b593967ba9e7bb2c7404390e76), you can see in red the deletion of information about GitHub data protection officer.
```
